.github/workflows/pr_release.yaml

```diff
@@ -0,0 +1,139 @@
+name: Build images on pull requests
+
+on: [pull_request]
+
+concurrency:
+  group: release-images-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-executor:
+    env:
+      GITHUB_SHA: ${{ github.sha }}
+      GITHUB_REF: ${{ github.ref }}
+      PLATFORMS: "linux/amd64,linux/arm64,linux/s390x,linux/ppc64le"
+    runs-on: ubuntu-latest
+    steps:
+    - name: Clone source code
+      uses: actions/checkout@v2
+
+    - name: Get the tags
+      id: vars
+      run: echo ::set-output name=tag::${GITHUB_REF/refs\/tags\//}
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+      with:
+        platforms: ${{ env.PLATFORMS }}
+
+    - name: Cache Docker layers
+      uses: actions/cache@v2
+      with:
+        path: /tmp/.buildx-cache
+        key: ${{ runner.os }}-buildx-${{ github.sha }}
+        restore-keys: |
+          ${{ runner.os }}-buildx-
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+      with:
+        version: latest
+       
+
+    - uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./deploy/Dockerfile_slim
+        platforms: linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
+        tags: |
+          gcr.io/kaniko-project/executor:${{ env.GITHUB_SHA }}-slim
```

Comment:
We definitely shouldn't be pushing to this project on PRs 🤔 

cc @dlorenc

Reply:
yeah push as well as signing is turned off now for project PRs.